iothub/device/src/IotHubBaseClient.cs

```diff
@@ -288,6 +306,7 @@ public async Task SendTelemetryAsync(IEnumerable<TelemetryMessage> messages, Can
         /// </remarks>
         /// <param name="directMethodCallback">The callback to be invoked when any method is invoked by the cloud service.</param>
         /// <param name="cancellationToken">A cancellation token to cancel the operation.</param>
+        /// <exception cref="InvalidOperationException">Thrown if the client instance is not opened already.</exception>
```

Comment:
These exception types were missing from the doc comments

Reply:
```suggestion
        /// <exception cref="InvalidOperationException">If the client instance is not already open.</exception>
```